docker-deploy/README.md

```diff
@@ -9,7 +9,9 @@ The nodes (target nodes) to install FATE must meet the following requirements:
 1. A Linux host
 2. Docker: 18+
 3. Docker-Compose: 1.24+
-4. Network connection to Internet to pull container images from Docker Hub. If network connection to Internet is not available, consider to set up [Harbor as a local registry](../registry/README.md) or use [offline images](https://github.com/FederatedAI/FATE/tree/master/docker-build).
+4. The deployment machine can be connected to the Internet, so the hosts can communicate with each other;
+5. Network connection to Internet to pull container images from Docker Hub. If network connection to Internet is not available, consider to set up [Harbor as a local registry](../registry/README.md) or use [offline images](https://github.com/FederatedAI/FATE/tree/master/docker-build).
```

Comment:
> https://github.com/FederatedAI/FATE/tree/master/docker-build
This link is 404.

I have also optimized this doc in branch 1.9.0. So if you submit this pr to master then there will be a conflict to handle when we merge master to 1.9.0, could you please also handle this?